types.cc

```diff
@@ -1883,7 +1883,8 @@ static uint32_t deserialize_value(const simple_date_type_impl&, bytes_view v) {
     return read_simple_exactly<uint32_t>(v);
 }
 
-static int64_t deserialize_value(const time_type_impl&, bytes_view v) {
+template<FragmentedView View>
+int64_t deserialize_value(const time_type_impl&, View v) {
```

Comment:
You can use the abbreviated syntax: `int64_t deserialize_value(const time_type_impl&, FragmentedView auto v)`, if you like.

Reply:
Wow, yet another concept syntax. I didn't know that was a thing.

> If you like

 I don't know. Should I like?

Reply:
> Should I like?

Good question. One the one hand, it has less fluff. On the other hand, it hides the fact that it is a template. On the feet, we may learn to recognize the syntax in time, and perhaps the fact that it is a template is not so important.